packages/components/src/Toast/Toast.stories.tsx

```diff
@@ -27,9 +28,9 @@ Alert.args = {
   color: "alert",
 };
 
-export const Dismissable = Template.bind(null);
-Dismissable.args = {
+export const NotDismissable = Template.bind(null);
+NotDismissable.args = {
   ...defaultArgs,
   color: "success",
-  onDismiss: () => console.log("dismissed!"),
+  onDismiss: undefined,
```

Comment:
If we do tie into the Storybook actions addon with the default args, I think we'd unfortunately have to make this `null`.

Otherwise the actions addon thinks we haven't passed anything and it should pass its own handler.